### ISSUE TITLE
an error reported using kera neural network model 

### ISSUE THREAD

**Issue by zh-zhang1984** [markdumke/reinforcelearn#36]
library(reinforcelearn)
library(keras)
library(tidyverse)
model = keras_model_sequential()
model %>% layer_dense(20, input_shape = 10, activation = "relu")
model %>% layer_dense(4, activation = "softmax")
keras::compile(model, loss = "mae", optimizer = keras::optimizer_sgd(lr = 0.4))

val = makeValueFunction("neural.network", model = model)
policy = makePolicy("epsilon.greedy", epsilon = 0.2)
algorithm = makeAlgorithm("qlearning")
agent = makeAgent("epsilon.greedy", 
                  val.fun = val,
                  val.fun.args = list(model=model), 
                  algorithm.args = list(class="qlearning"), 
                  policy.args = list(epsilon = 0.2))

Error in makeAgent("epsilon.greedy", val.fun = val, val.fun.args = list(model = model),  : 
  Assertion on 'model' failed: Must have class 'keras.models.Sequential', but has classes 'keras.engine.sequential.Sequential','keras.engine.training.Model','keras.engine.network.Network','keras.engine.base_layer.Layer','tensorflow.python.training.checkpointable.base.CheckpointableBase','python.builtin.object'.

How can I solve the problem?
